mobile/app/src/main/AndroidManifest.xml

```diff
@@ -43,29 +43,25 @@
             <!-- Handles the implicit intent to VIEW the intel.ingress.com/ URI -->
             <intent-filter>
                 <action android:name="android.intent.action.VIEW"/>
-                <action android:name="android.nfc.action.NDEF_DISCOVERED"/>
```

Comment:
This is not required for NFC to work? I don't have the proper device to test

Reply:
I'm playing around trying to figure why some apps are fine with urls, and why iitc isn't